### PR TITLE
#240 update ground floor height term "method" to "gfh_method"

### PR DESCRIPTION
**Pull request by Santonia27**
-  update ground floor height term "method" to "gfh_method"
-  update ground floor height term "attr_name" to "attribute_name"

### CODE REVIEW

File: hydromt_fiat/fiat.py [hunk 1]
```diff
@@ -425,7 +425,7 @@ def update_max_potential_damage(
         damage_types: Union[List[str], str, None] = None,
         country: Union[str, None] = None,
         target_attribute: Union[str, List[str], None] = None,
-        attr_name: Union[str, List[str], None] = None,
+        attribute_name: Union[str, List[str], None] = None,
         method: Union[str, List[str], None] = "nearest",
```
[the remark this answers is not in the document]
Comment (Santonia27) [reply hydromt_fiat/fiat.py:429]: Damages are updated in a different branch

File: hydromt_fiat/fiat.py [hunk 2]
```diff
@@ -435,7 +435,7 @@ def update_max_potential_damage(
                 damage_types=damage_types,
                 country=country,
                 target_attribute=target_attribute,
-                attr_name=attr_name,
+                attribute_name=attribute_name,
                 method=method,
```
[the remark this answers is not in the document]
Comment (Santonia27) [reply hydromt_fiat/fiat.py:439]: Damages are updated in a different branch

File: hydromt_fiat/workflows/exposure_vector.py [hunk 3]
```diff
@@ -590,7 +593,7 @@ def setup_max_potential_damage(
             int, float, str, Path, List[str], List[Path], pd.DataFrame
         ] = None,
         damage_types: Union[List[str], str, None] = None,
-        attr_name: Union[str, List[str], None] = None,
+        attribute_name: Union[str, List[str], None] = None,
         method: Union[str, List[str], None] = "nearest",
```
[the remark this answers is not in the document]
Comment (Santonia27) [reply hydromt_fiat/workflows/exposure_vector.py:597]: Damages are updated in a different branch

File: hydromt_fiat/workflows/exposure_vector.py [hunk 4]
```diff
@@ -684,9 +687,9 @@ def setup_max_potential_damage(
             # assign the ground floor height to the assets
             gfh = self.data_catalog.get_geodataframe(max_potential_damage)
             gdf = self.get_full_gdf(self.exposure_db)
-            gdf = join_spatial_data(gdf, gfh, attr_name, method, max_dist, self.logger)
```
[the remark this answers is not in the document]
Comment (Santonia27) [reply hydromt_fiat/workflows/exposure_vector.py:687]: Damages are updated in a different branch

File: hydromt_fiat/workflows/exposure_vector.py [hunk 5]
```diff
@@ -527,8 +527,8 @@ def setup_aggregation_labels(self):
     def setup_ground_floor_height(
         self,
         ground_floor_height: Union[int, float, None, str, Path, List[str], List[Path]],
-        attr_name: Union[str, List[str], None] = None,
-        method: Union[str, List[str], None] = "nearest",
+        attribute_name: Union[str, List[str], None] = None,
+        gfh_method: Union[str, List[str], None] = "nearest",
```
[the remark this answers is not in the document]
Comment (Santonia27) [reply hydromt_fiat/workflows/exposure_vector.py:531]: done

**Review by dladrichem — APPROVED** [review]

**Pull request merged**